### PR TITLE
Update prices for AWS

### PR DESCRIPTION
**Pull request by tomislater**

### CODE REVIEW

**Comment by tomislater** [comment]
I have gotten the results by running `python kube_resource_report/pricing.py`.

File: kube_resource_report/pricing.py [hunk 1]
```diff
@@ -144,7 +145,7 @@ def generate_ec2_price_list():
     max_price = {}
     for location in sorted(LOCATIONS.values()):
         # some regions are not available
-        if location in ("ap-northeast-3", "cn-north-1", "cn-northwest-1", "us-gov-west-1", "us-gov-east-1"):
+        if location in ("ap-northeast-3", "cn-north-1", "cn-northwest-1", "us-gov-west-1", "us-gov-east-1", "ap-east-1"):
```
Comment (hjacobs) [inline kube_resource_report/pricing.py]: Why did you add "ap-east-1" to the list? Looks like you could get the prices for this region..

Comment (tomislater) [reply kube_resource_report/pricing.py]: I cannot...

```
botocore.exceptions.ClientError: An error occurred (AuthFailure) when calling the DescribeSpotPriceHistory operation: AWS was not able to validate the provided access credentials
```

You have to ask for permission for this region. You do not have access there by default.

Comment (hjacobs) [reply kube_resource_report/pricing.py]: Yes, but EC2 prices are updated, maybe only spot is a problem

**Comment by tomislater** [comment]
I have just added this region because it was missed in the file. What do you propose?

File: kube_resource_report/pricing.py (hunk 1 above)
Comment (hjacobs) [reply kube_resource_report/pricing.py]: Sorry, forget what I said :-/

**Comment by hjacobs** [comment]
@tomislater all good, I was confused with my own code.

**Comment by hjacobs** [comment]
Thanks!
